content/en/blog/2021/etcd-deployments-on-AWS.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Deployment of etcd on AWS EC2
+author: Somoshree Datta
+date: 2021-09-03
+---
+
+This blog post provides an introduction to design considerations when designing an etcd deployment on AWS EC2 and how AWS specific features may be utilized in that context. Also, this post assumes operational knowledge of Amazon Web Services (AWS), specifically Amazon Elastic Compute Cloud (EC2). This blog has been organized as follows:
+
+* [Capacity Planning](#capacity-planning)
+* [Cluster Design](#cluster-design)
+* [Availability](#availability)
+* [Data durability after member failure](#data-durability-after-member-failure)
+* [Perfomance/Throughput](#performancethroughput)
+* [Network](#network)
+* [Disk](#disk)
+* [Self-healing](#self-healing)
+* [Next Steps](#next-steps)
+
+## Capacity Planning
+
+As a critical building block for distributed systems it is crucial to perform adequate capacity planning in order to support the intended cluster workload. As etcd is a highly available and strongly consistent data store, so increasing the number of nodes in an etcd cluster will generally affect performance adversely. This makes sense intuitively, as more nodes means more members for the leader to coordinate state across. The most direct way to increase throughput and decrease latency of an etcd cluster is allocate more disk I/O, network I/O, CPU, and memory to the cluster members. In this scenario, it is impossible to temporarily divert incoming requests to the cluster, though scaling the EC2 instances which comprise the etcd cluster members one at a time may improve performance. It is, however, best to avoid bottlenecks through capacity planning.
+
+The etcd team has produced a [hardware recommendation guide](https://etcd.io/docs/v3.5/op-guide/hardware/) which is very useful for "ballparking" how many nodes and what instance type are necessary for a cluster.
```

Comment:
It's a bit more unweildly, but since his is an internal links to the site, I'd recommend using a relative link here.
```suggestion
The etcd team has produced a [hardware recommendation guide](../../../docs/v3.5/op-guide/hardware/) which is very useful for "ballparking" how many nodes and what instance type are necessary for a cluster.
```